File: src/main/java/com/history/tree/model/Person.java
package com.history.tree.model;


import lombok.Getter;
import lombok.Setter;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

import java.time.LocalDate;

@Getter
@Setter
@Table("person")
public class Person {

    @Id
    @Column("id")
    private Long id;

    @Column("first_name")
    private String firstName;

    @Column("last_name")
    private String lastName;

    @Column("patronymic")
    private String patronymic;

    @Column("tree_id")
    private Long treeId;

    @Column("gender")
    private Character gender;

    @Column("birth_date")
    private LocalDate birthDate;

    @Column("date_of_death")
    private LocalDate dateOfDeath;

}

File: src/main/kotlin/com/history/tree/model/Person.kt
package com.history.tree.model

import org.springframework.data.annotation.Id
import org.springframework.data.annotation.Version
import org.springframework.data.relational.core.mapping.Column
import org.springframework.data.relational.core.mapping.Table
import java.time.LocalDate
import java.util.*

@Table("person")
data class Person (
    @Id
    @Column("id")
    val id: UUID,

    @Column("first_name")
    var firstName: String?,

    @Column("last_name")
    var lastName: String?,

    @Column("patronymic")
    var patronymic: String?,

    @Column("tree_id")
    var treeId: UUID,

    @Column("gender")
    var gender: Char,

    @Column("birth_date")
    var birthDate: LocalDate?,

    @Column("date_of_death")
    var dateOfDeath: LocalDate?,
)